test/release/examples/benchmarks/ssca2/SSCA2_Modules/analyze_torus_graphs_stencil_rep_v2.chpl

```diff
@@ -69,10 +69,11 @@ module analyze_torus_graphs {
     writeln ( " dimension  lower  upper" );
     writeln ( "            bound  bound" );
 
-    for d in 1 .. G.dimensions do 
-      writeln ( format ("########", d), 
-		format ("########", vertex_domain.dim(d).low),  
-		format ("########", vertex_domain.dim(d).high) );
+    for d in 1 .. G.dimensions {
+      writef ("########", d);
+      writef ("########", vertex_domain.dim(d).low);  
+      writef ("########\n", vertex_domain.dim(d).high);
```

Comment:
Good catch!  (And, one less conversion required if we go with the %{} required approach...)